project-fortis-interfaces/src/components/Insights/Maps/HeatMap.js

```diff
@@ -93,7 +98,7 @@ export default class HeatMap extends React.Component {
   }
 
   getMapBounds(bbox) {
-    return bbox.length && bbox.length === 4 ? [[bbox[1], bbox[0]], [bbox[3], bbox[2]]] : [];
+    return bbox.length && bbox.length === 4 ? [[bbox[0], bbox[1]], [bbox[2], bbox[3]]] : [];
```

Comment:
Please also refactor the other places in this component to use this method so that for sure all the bounding box conversions are correct, e.g.:

https://github.com/CatalystCode/project-fortis/blob/426df52ff19f8efcd4468f30c0553267f53683b9/project-fortis-interfaces/src/components/Insights/Maps/HeatMap.js#L21

https://github.com/CatalystCode/project-fortis/blob/426df52ff19f8efcd4468f30c0553267f53683b9/project-fortis-interfaces/src/components/Insights/Maps/HeatMap.js#L112

https://github.com/CatalystCode/project-fortis/blob/426df52ff19f8efcd4468f30c0553267f53683b9/project-fortis-interfaces/src/components/Insights/Maps/HeatMap.js#L119

https://github.com/CatalystCode/project-fortis/blob/426df52ff19f8efcd4468f30c0553267f53683b9/project-fortis-interfaces/src/components/Insights/Maps/HeatMap.js#L127